buildpack.md

```diff
@@ -209,21 +213,19 @@ If a cache is provided the lifecycle:
 - SHOULD store the Layer Content Metadata associated with each layer so that it can be recovered using the layer Diff ID
 
 Before the next re-build:
-- The lifecycle:
-  - MUST either restore the entire `<layers>/<layer>.toml` file and corresponding `<layers>/<layer>/` directory from any previous build to the same paths or
-  - MUST restore neither the `<layers>/<layer>.toml` file nor corresponding `<layers>/<layer>/` directory.
+- The lifecycle MUST do both or neither of the following:
```

Comment:
Is it worth calling out that "both" is for `cache = true` and "neither" is for `cache = false`?

Reply:
Also there is that edge case where `cache = false` and `launch = true`. I don't know if that's too much detail...

Reply:
I think it's important that the spec will be clear enough and explain all of the cases. I'll try to make it clearer.